### PR TITLE
[App Service] `az staticwebapp create`: Allow creating Static Web Apps not connected to a github repo

### LINKED ISSUE #21058: StaticWebApp:az staticwebapp create - skip GitHub workflow generation
Bit of feedback for the CLI team - az staticwebapp create should have an option to NOT create the GitHub action workflow file, in case the user is using Azure Pipelines or another model.  Thanks!

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: e683c181-f005-a6c8-cb11-74ce557e51f8
* Version Independent ID: 860411b7-f2f1-af2b-b5e2-7d67e1b8e028
* Content: [az staticwebapp](https://docs.microsoft.com/en-us/cli/azure/staticwebapp?view=azure-cli-latest)
* Content Source: [latest/docs-ref-autogen/staticwebapp.yml](https://github.com/MicrosoftDocs/azure-docs-cli/blob/main/latest/docs-ref-autogen/staticwebapp.yml)
* GitHub Login: @rloutlaw
* Microsoft Alias: **routlaw**

### PR DESCRIPTION
**Pull request by StrawnSC**
**Description**

Fixes #21058

**Testing Guide**

`az staticwebapp create -g {} -n {}`

**History Notes**

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [x] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by panchagnula** [comment]
@anthonychu & @simonaco FYI

File: src/azure-cli/azure/cli/command_modules/appservice/static_sites.py [hunk 1]
```diff
@@ -364,8 +364,8 @@ def update_staticsite_users(cmd, name, roles, authentication_provider=None, user
                                           static_site_user_envelope=user_envelope)
 
 
-def create_staticsites(cmd, resource_group_name, name, location,  # pylint: disable=too-many-locals,
-                       source, branch, token=None,
+def create_staticsites(cmd, resource_group_name, name, location="centralus",  # pylint: disable=too-many-locals,
```
Comment (panchagnula) [inline src/azure-cli/azure/cli/command_modules/appservice/static_sites.py:367]: Did we want to make location optional & Default to CentralUs?

**Review by panchagnula — APPROVED** [review]
LGTM . Given there is no need for GitHub integration we can probably create a test case for empty SWA

File: src/azure-cli/azure/cli/command_modules/appservice/static_sites.py (hunk 1 above)
Comment (wangzelin007) [reply src/azure-cli/azure/cli/command_modules/appservice/static_sites.py:367]: > Did we want to make location optional & Default to CentralUs?

Hi [StrawnSC](https://github.com/StrawnSC),
Can you confirm with this ?

**Review by wangzelin007 — APPROVED** [review]

File: src/azure-cli/azure/cli/command_modules/appservice/static_sites.py (hunk 1 above)
Comment (StrawnSC) [reply src/azure-cli/azure/cli/command_modules/appservice/static_sites.py:367]: Yes, this is what @simonaco put on the ADO work item

**Review by zhoxing-ms — APPROVED** [review]
